---

spiral_lc_node:
  ros__parameters:
    type: node
    modes:
      __DEFAULT__:
        ros__parameters:
          spiral_width: 1.0
          spiral_altitude: 1.0
      FAST:
        ros__parameters:
          spiral_width: 1.0
          spiral_altitude: 2.0
      SLOW:
        ros__parameters:
          spiral_width: 1.0
          spiral_altitude: 0.5

thruster:
  ros__parameters:
    type: node
    modes:
      __DEFAULT__:
        ros__parameters:
          thruster1: 1
          thruster2: 1
          thruster3: 1
          thruster4: 1
          thruster5: 1
          thruster6: 1
      TF1:
        ros__parameters:
          thruster1: 0
          thruster2: 1
          thruster3: 1
          thruster4: 1
          thruster5: 1
          thruster6: 1
      TF2:
        ros__parameters:
          thruster1: 1
          thruster2: 0
          thruster3: 1
          thruster4: 1
          thruster5: 1
          thruster6: 1
      TF3:
        ros__parameters:
          thruster1: 1
          thruster2: 1
          thruster3: 0
          thruster4: 1
          thruster5: 1
          thruster6: 1
      TF4:
        ros__parameters:
          thruster1: 1
          thruster2: 1
          thruster3: 1
          thruster4: 0
          thruster5: 1
          thruster6: 1
      TF5:
        ros__parameters:
          thruster1: 1
          thruster2: 1
          thruster3: 1
          thruster4: 1
          thruster5: 0
          thruster6: 1
      TF6:
        rot__parameters:
          thruster1: 1
          thruster2: 1
          thruster3: 1
          thruster4: 1
          thruster5: 1
          thruster6: 0

